migrator/migrations/n_52_to_n_53_postgres_simple_access_scopes/legacypermissionsets/rocksdb_plugin.go

```diff
@@ -0,0 +1,59 @@
+// Code generated by rocksdb-bindings generator. DO NOT EDIT.
```

Comment:
Since you copied this file, it is no longer generated.  We should probably update the comment.